src/hotspot/share/gc/z/zReferenceProcessor.hpp

```diff
@@ -41,6 +41,9 @@ class ZReferenceProcessor : public ReferenceDiscoverer {
 
   ZWorkers* const      _workers;
   ReferencePolicy*     _soft_reference_policy;
+  // Judge whether the _soft_reference_policy is AlwaysClearPolicy or not,
+  // use this extra field to avoid using RTTI (runtime type information).
```

Comment:
This comment confuses me more than it helps. I don't care what the type of the current soft reference policy is, as much as I care about its behaviour. Using RTTI to determine this behaviour based on the type of the current soft ref policy is just one alternative implementation for inspecting the behaviour. I don't think it needs to be mentioned.

Reply:
Fixed. I remove the comment.